### PR TITLE
Fixed GitHub detection of license in LICENSE.txt file.

### PR DESCRIPTION
**Pull request by uldall**
Unfortunately licensee also looks at the COPYRIGHT.txt file and detects it as "other".
GitHub therefore still doesn't add the correct license to the project. This is
however a step in the right direction.

Before:
$ licensee LICENSE.txt
License: Other
Matched files: ["LICENSE.txt"]
LICENSE.txt:
  Content hash: 8dd7a8f24846f434ce7fdb96f6440909b5469277
  Attribution: Copyright (c) 2007-2017 Juan Linietsky, Ariel Manzur.
  License: Other

After:
$ licensee LICENSE.txt
License: MIT License
Matched files: ["LICENSE.txt"]
LICENSE.txt:
  Content hash: d64f3bb4282a97b37454b5bb96a8a264a3363dc3
  Attribution: Copyright (c) 2007-2017 Juan Linietsky, Ariel Manzur.
  Confidence: 100.00%
  Matcher: Licensee::Matchers::Exact
  License: MIT License

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -18,5 +18,3 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
-
--- Godot Engine <https://godotengine.org>
```
Comment (akien-mga) [inline LICENSE.txt]: :'(

**Comment by akien-mga** [comment]
It worked a few months ago... They should loosen their parsing instead of tightening it, the addition of an URL after the license obviously does not invalidate it.

**Comment by akien-mga** [comment]
Still doesn't work as it used to in #8682.

**Comment by uldall** [comment]
Yeah, they have added some logic when they find both a LICENSE.txt and a COPYRIGHT.txt which breaks it. See here: https://github.com/benbalter/licensee/pull/203

**Pull request merged**